ddtrace/propagation/http.py

```diff
@@ -950,6 +966,18 @@ def parent_call():
             for key in span_context._baggage:
                 headers[_HTTP_BAGGAGE_PREFIX + key] = span_context._baggage[key]
 
+        if hasattr(ddtrace, "tracer") and hasattr(ddtrace.tracer, "sample"):
```

Comment:
We should log an error if this if statement returns False, these fields should always be set